src/Microsoft.Identity.Abstractions/TokenAcquisition/AcquireTokenOptions.cs

```diff
@@ -90,6 +91,42 @@ public AcquireTokenOptions(AcquireTokenOptions other)
         /// </summary>
         public string? PopClaim { get; set; }
 
+        /// <summary>
+        /// When <see cref="ManagedIdentity"/> is set, the application uses a managed identity instead of client credentials to
+        /// acquire an app token.<br/><br/>
+        /// 
+        /// The type of a managed identity is defined by the <see cref="ManagedIdentityDescription.IdType"/> field, when using a
+        /// <see cref="ManagedIdentityType.SystemAssigned"/> identity, this is the only field that needs to be set.<br/><br/>
```

Comment:
It actually does not need to be set if this is the default?

Reply:
It doesn't need to be this is from what I was asking around if we should model using the more readable solution even if it could be done with less code. i.e. if I look at an appsettings file and it just says managed identity I don't necessarily know there are different types of managed identities, nor would I know what type is being used without checking the documentation.